### DISCUSSION TITLE
Changing rpiCamera parameters

### DISCUSSION THREAD

**Discussion by RickRoz** [bluenviron/mediamtx#2796] [Questions]
### Question

Hello,

I have the mediamtx server up and running, which i have also added in Home Assistant with webRTC.
I wanted to change some configuration parameters in mediamtx but nothing changes in the live stream. Any idea how i can change the settings?

**Comment by aler9** [comment]
Hello, the server is able to detect changes in the configuration file and to apply them immediately. This is documented in the README (search for "hot reloading"). If you have problems, start the server with `logLevel: debug`, replicate the issue and attach a server log.

**Reply by RickRoz** [reply]
Hello Aler9,

So i tested it again. I stopped the server and changed some parameters in the nano mediamtx.yml file, i manually started to steam with "mediamtx". I saw the changes on the live stream. So i thought great, i used the command sudo systemctl start mediamtx to start on boot.

I went and checked the screen and the changes that i made are gone. 

How do i execute the loglevel: debug? I am sorry but still a rookie in regards to this.

Thanks!

[a comment by an automated account is not shown]
